sb3_contrib/common/wrappers/time_feature.py

```diff
@@ -0,0 +1,88 @@
+from typing import Dict, Union
+
+import gym
+import numpy as np
+from stable_baselines3.common.type_aliases import GymObs, GymStepReturn
+
+
+class TimeFeatureWrapper(gym.Wrapper):
+    """
+    Add remaining time to observation space for fixed length episodes.
```

Comment:
"Add remaining, normalized time..." (just to clarify that the number is already normalized so that networks won't throw a tantrum)